ovos_utils/fingerprinting.py

```diff
@@ -27,8 +28,12 @@ def detect_platform():
 
 def get_config_fingerprint(config=None):
     if not config:
-        from ovos_config.config import Configuration
-        config = Configuration()
+        try:
+            from ovos_config.config import read_mycroft_config
+            config = read_mycroft_config()
```

Comment:
For a future release, consider making `config` a required arg and dropping ovos-config fallback

Reply:
I wasn't sure if it was being used anywhere, I can just log deprecation warnings in all the functions though and we can drop it in 0.1?